src/LOAuth.ts

```diff
@@ -117,7 +117,7 @@ class LOAuth {
    * The cookie can be read by another application on a compatible domain by triggering
    * refreshAuthToken, which will find and import the cookie as a token
    */
-  public setDomainCookieAuthState(token: LOAuth.CookieTokenData) {
+  public static setDomainCookieAuthState(token: LOAuth.CookieTokenData) {
```

Comment:
This makes it super simple for phc-login to use this without fully adopting this library yet.